apis/spaces/v1beta1/conditions.go

```diff
@@ -53,6 +53,18 @@ const (
 
 	// ReasonRestorePending indicates that the control plane restore is pending.
 	ReasonRestorePending xpcommonv1.ConditionReason = "RestorePending"
+
+	// ConditionTypeRunning indicates whether the workloads on the Control Plane
+	// are running or not.
+	ConditionTypeRunning xpcommonv1.ConditionType = "Running"
```

Comment:
Also the comment is misleading, should be limited to Crossplane only.

```suggestion
	ConditionTypeRunning xpcommonv1.ConditionType = "CrossplaneRunning"
```

Reply:
Done!